guide/C/ch_invest.xml

```diff
@@ -1817,6 +1817,1129 @@ Income
         </mediaobject>
       </screenshot>
     </sect2>
+    
+    <sect2 id="invest-sellLots">
+      <title>Automatic Calculation of Capital Gains or Losses Using Lots</title>
+       
+      <para><application>&app;</application> can automatically calculate and
+        create the capital gain or loss transactions for security sales. Lots
+        are used to link particular sell transaction splits to the buy 
+        transaction splits so that the actual cost of the sold securities can be
+        used to calculate the gain or loss.
+      </para>
+
+      <para>If you make an error, you can delete the lot(s) and/or delete the 
+        automatically created capital gain/loss transaction(s) and retry.
+      </para>
+      
+      <para><application>&app;</application> can automatically create lots or a
+        user may create lots. <application>&app;</application> can automatically
+        link buy and sell transaction splits to lots or a user may do the 
+        linking.
+      </para>
+
+      <para>
+        <application>&app;</application> can automatically <emphasis>scrub
+        </emphasis> a security account. <application>&app;</application>
+        scrubbing, when referring to investments, involves creating lots
+        linking sell transaction splits to buy transaction splits, and then 
+        creating a gain/loss transaction for the difference between the sale
+        value and the cost of the securities sold. This can be a useful feature
+        reducing manual calculation especially if dividends have been
+        reinvested over years and there are many different costs involved
+        in a sale.
+      </para>
+
+      <para>The term <emphasis>scrub</emphasis> is used because security
+        accounts need to be cleaned up after sales to ensure the 
+        difference between the cost paid for securities, and value
+        received from selling them, is accounted for as capital gain or
+        loss. If the capital gain/loss is not correct, the Trial Balance
+         (<menuchoice>
+            <guimenu>Reports</guimenu>
+            <guisubmenu>Income &amp; Expense</guisubmenu>
+            <guimenuitem>Trial Balance</guimenuitem>
+          </menuchoice>) bottom line total debits will not balance to 
+          total credits.
+      </para>
+                    
+      <sect3 id="invest-sellLimitations">
+        <title>Limitations</title>
+        
+        <para>There are some limitations to using this feature.</para>
+        
+        <itemizedlist>
+          <listitem>
+            <para><application>&app;</application> has only 1 method of
+              automatically linking buy transactions to sell transactions -
+              First In/ First Out (FIFO). I.e. The oldest securities are always
```

Comment:
I have changed this to:
GnuCash implements only the First In/ First Out (FIFO) cost method when automatically linking buy transactions to sell transactions.

I feel it is important to mention automatic linking because you can use lots to do LIFO but not without manually doing the linking. I haven't mentioned LIFO here as that is mentioned later. If you'd like me to remove the reference to automatic linking, I think I should mention LIFO here too.